R/linter_tags.R

```diff
@@ -142,11 +142,13 @@ available_tags <- function(packages = "lintr") {
 #' @param linter_name Name of the linter to generate Rd code for.
 #'
 #' @noRd
-rd_tags <- function(linter_name) {
```

Comment:
This change seems unnecessary to me.

Reply:
fair, reverted in favor of just linking `[deprecated_linters]`